src/igvfd/searches/configs/technical_sample.py

```diff
@@ -54,10 +72,27 @@ def technical_sample():
             {
                 'title': 'Sample',
                 'facet_fields': [
+                    'taxa',
+                    'classifications',
                     'sample_terms.term_name',
+                    'sample_material',
                     'virtual',
-                    'classifications',
-                    'file_sets.assay_term.term_name',
+                ]
+            },
+            {
+                'title': 'File Set',
+                'facet_fields': [
+                    'file_sets.@type',
+                    'file_sets.preferred_assay_title',
+                    'file_sets.file_set_type',
+                ]
+            },
+            {
+                'title': 'Construct',
+                'facet_fields': [
+                    'construct_library_sets.file_set_type',
+                    'construct_library_sets.associated_phenotypes.term_name',
+                    'nucleic_acid_delivery',
```

Comment:
I think it is missing the "construct_library_sets." at the beginning?

Reply:
This property is on the sample, not construct library set